src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidAccountManagerBroker.cs

```diff
@@ -448,5 +448,10 @@ private void CheckPackageForPowerOptimization(string package)
                        null,
                        UiRequiredExceptionClassification.AcquireTokenSilentFailed);
         }
+
+        public Task<MsalTokenResponse> AcquireTokenByUsernamePasswordAsync(AuthenticationRequestParameters authenticationRequestParameters, AcquireTokenByUsernamePasswordParameters acquireTokenByUsernamePasswordParameters)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
Why do you throw on Android but not on iOS? Should be consistent. Probably return null to fallback to MSAL's default logic.

Reply:
Added logic to return null to fallback to msal's default logic.